### PR TITLE
restructuring installation topics for EDB*Plus

### PR DESCRIPTION
**Pull request by dwicinas**
Create a hierarchical structure based on architecture for Linux installation topics for EDB*Plus

### CODE REVIEW

**Comment by dwicinas** [comment]
@drothery-edb This is ready for review. I made some decisions on how to structure non-standard information (prerequisites and post-install configuration steps). Happy to reorganize if you have suggestions.

File: product_docs/docs/edb_plus/40/03_installing_edb_plus/install_on_linux/x86_amd64/07_edbplus_ubuntu20_deb10_x86.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+1. Configure the EDB repository:
+
+    On Ubuntu 20:
+    ```text
+    sh -c 'echo "deb https://USERNAME:PASSWORD@apt.enterprisedb.com/$(lsb_release -cs)-edb/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/edb-$(lsb_release -cs).list'
+    ```
+    On Debian 10:
+    
+    a.  Set up the EDB repository:
+        ```text
```
Comment (drothery-edb) [inline product_docs/docs/edb_plus/40/03_installing_edb_plus/install_on_linux/x86_amd64/07_edbplus_ubuntu20_deb10_x86.mdx:26]: the code block isn't rendering quite right - the tick marks are included in it. Same thing with the following code block. Not sure how much time we want to spend fixing it.

**Review by drothery-edb — COMMENTED** [review]
couple small formatting issues - otherwise, I think it is fine.

**Comment by dwicinas** [comment]
@drothery-edb I fixed the issue of the errant tick marks. Also did a little work on the nav bar. Should be ready for review again.

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**